collections/bitcoin/_posts/2022-10-02-bitcoin-is-digital-scarcity.md

```diff
@@ -376,7 +376,7 @@ sentence to the lore surrounding the ring: *"The Stone of Jordan is far
 more valuable than its appearance would suggest. Men have given much to
 possess it."*
 
-Once people hone in on one form of money, they do indeed go through
+Once people home in on one form of money, they do indeed go through
```

Comment:
[Today I learned!](https://www.merriam-webster.com/words-at-play/home-in-or-hone-in)

Reply:
I had to double-check that myself  ;)